### DISCUSSION TITLE
Let Us Open A Chat Window

### DISCUSSION THREAD

**Discussion by skeddles** [community/community#42878] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I frequently want to use copilot to debug a few certain lines, but it's not really good at correcting stuff that's already there.

Instead i have to open up ChatGPT and paste my code in and explain it (and im guessing ChatGPT will stop being free soon, so im hoping there will be a replacement)

It would be so much better if I could click a button within VSCode and it opens a chatgpt window which i can can ask it questions pertaining to my code, and have it suggest fixes, instead of trying to trick it into writing the correct code with comments.

**Comment by skeddles** [comment]
![image](https://user-images.githubusercontent.com/2721332/210440060-da0beaa6-a77b-488f-a943-b86e0852bafb.png)

perfect example, i had to ask a question and it walked me through what the problem could be, and how to understand it better

**Comment by cmp-nct** [comment]
Copilot is by far too bad for something like a chat window.
You clearly want it to behave like ChatGPT, it would just repeat useless sentences over and over.
Copilot is a full generation too old to be that good, it got surpassed by "GPT 3.5" within months of release.

If you look at the code GPT 3.5 writes, that's extremely much better than the stuff copilot writes.

**Reply by pixelomo** [reply]
[oh how the turntables]( https://twitter.com/swyx/status/1638550858073006089?s=20)

**Reply by cmp-nct** [reply]
Yea it was obvious they have to do something, though they are really slow with innovations compared to other projects. 
I don't think the Copilot team has a lot of attention.
Currently it's just a waitlist for a demo and I've not heard about anyone actually getting activated. 
In any case .. it sounds like a very good innovation. It just doesn't sound like it's available soon.

**Reply by skeddles** [reply]
I did it! you're welcome everyone
